### ISSUE TITLE
How to use in where queries

### ISSUE THREAD

**Issue by plandolt** [darsyn/ip#91]
I cannot manage to create a working query filtering based on the ip. I figure the problem is in packing and unpacking the binary before saving in the database. Or am I missing something?

**Comment by zanbaldwin** [comment]
Yes, in order to use with the query builder, you'll have to convert to the underlying binary string. This is a limitation I couldn't get around when I first wrote the Doctrine integration, and haven't really addressed it since then.

However, I just spun up a quick project to test out the response I _was_ going to write, but I ran into loads of errors and incompatibilities with the newer DBAL4/ORM3 packages, so it looks like I'm going to be rewriting the Doctrine part now anyway :woman_shrugging:

I'll see if I can figure out how to pass IP objects to query builders and findBy repository methods while I'm fixing everything else!
